src/database.ts

```diff
@@ -187,6 +224,7 @@ export function getSettings(): Knex.Config {
       loadExtensions: ['.js'],
     },
     pool,
+    seeds: isDevelopment ? seedInfo : undefined,
```

Comment:
It's ok for seeds to always be in here, but they shouldn't always run. So this doesn't need to be conditional.

Reply:
So far these are only supposed to run IF the env variables are set and on development

Reply:
Seed files should not automatically run, so this conditional is not useful. Keeping `seeds` always set here should not have an effect whether we're in dev or production mode.